index.html

```diff
@@ -129,5 +129,383 @@ <h2 data-dft-for="">
         "https://wicg.io">WICG</a>.
       </p>
     </section>
+    <section data-cite="DOM">
+      <h2>
+        Installable web applications
+      </h2>
+      <section>
+        <h2>
+          Installation prompts
+        </h2>
+        <p>
+          There are multiple ways that the installation process can be
+          triggered:
+        </p>
+        <ul>
+          <li>An end-user can <dfn data-lt="manual installation">manually</dfn>
+          trigger the installation process through the user agent's
+            <abbr title="User Interface">UI</abbr>, directly invoking the steps
+            to <a>present an install prompt</a>.
+          </li>
+          <li>The installation process can occur through an <dfn>automated
+          install prompt</dfn>: that is, a UI that the user agent presents to
+          the user when, for instance, there are sufficient <a data-cite=
+          "appmanifest#installability-signals">installability signals</a> to
```

Comment:
I removed the "installability signals" from the other spec, as they turned out to be somewhat Chrome specific. So it might be good to just say:

```suggestion
          the user when, for instance, there are sufficient signals to
```